packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -217,11 +205,13 @@ class WebpackCLI extends GroupHelper {
      * @returns {void}
      */
     _handleDefaultEntry() {
-        if (!this.basicGroup) {
-            const BasicGroup = require('./groups/BasicGroup');
-            this.basicGroup = new BasicGroup();
+        let defaultEntry;
+        const rootIndexPath = path.resolve('index.js');
+        if (existsSync(rootIndexPath)) {
+            defaultEntry = './index.js';
+        } else {
+            defaultEntry = './src/index.js';
```

Comment:
Do we have tests for both of the default entries i.e, `./index.js` and `./src/index.js` ?

Reply:
This should be solved on webpack side, here invalid solution, because index can have `cjs`/`ts`/`mjs` and etc extensions

Reply:
Need to change a bunch of tests again because of current behaviour, let's fix this separately? Here's the issue for any discussion - https://github.com/webpack/webpack-cli/issues/1852